src/components/editMeasure/editor/MeasureEditor.tsx

```diff
@@ -481,6 +485,23 @@ const MeasureEditor = () => {
     setToastOpen(true);
   };
 
+  const handleEditLibrary = (
```

Comment:
unit test would be good to have. I can help you if you need help.

Reply:
thanks for the help. I just pushed the requested change